eng/common/scripts/get-markdown-files-from-changed-files.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+    # The root repo we scaned with.
+    [string[]] $RootRepo = "./"
+)
+$deletedFiles = (git diff origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH} HEAD --name-only --diff-filter=D)
```

Comment:
I think it is a good idea to accept the branch to diff against as argument to the script as well. We can default the value to this but I still think it is worth accepting as it would have made your local testing you commented out easier as well :)